recipes/hydride/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.0.0" %}
+{% set number = "0" %}
+{% set sha256 = "79f9a335dcffe060197a02b9a86f0b14d286dda881b5d1cf7d2db4c784951bfc" %}
+
+package:
+  name: hydride
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/hydride/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ number }}
+  script: {{ PYTHON }} -m pip install . --no-use-pep517 --no-deps --ignore-installed -vv
+  skip: true  # [py < 37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - setuptools >=30.0
+    - wheel >=0.30
+    - cython >=0.29
+    - numpy =1.14  # [py < 39]
+    - numpy =1.19  # [py >= 39]
```

Comment:
Is it not enough just to do `numpy` and let the build take care of what the appropriate version is?

Reply:
Doesn't this mean, the build will take the most recent version, if compatible? In this case this would mean that to my knowledge the runtime requirement needs to be at least that `numpy` version, disallowing the support for earlier `numpy` versions.

Reply:
When listed under `host`, it's going to take the one appropriate one from an ABI perspective (for packages that build against the NumPy C-API). Right now that's 1.18 or 1.19, depending on the Python version. Unless you have a specific need for an older NumPy, it'd be better to just let conda-forge's build pinning machinery handle this.

Reply:
Thank you for the explanation, I relaxed the dependency. From my side this PR is ready to be merged now,